nominal/core/client.py

```diff
@@ -1450,6 +1456,82 @@ def search_data_reviews(
 
         return [DataReview._from_conjure(self._clients, data_review) for data_review in raw_data_reviews]
 
+    def get_containerized_extractor(self, rid: str) -> ContainerizedExtractor:
+        return ContainerizedExtractor._from_conjure(
+            self._clients,
+            self._clients.containerized_extractors.get_containerized_extractor(self._clients.auth_header, rid),
+        )
+
+    def create_containerized_extractor(
```

Comment:
With this API, the user is responsible for understanding the various objects: `DockerImageSource`, `TagDetails`, and, perhaps hardest, the `TimestampMetadata`.

Could we instead allow them to construct those directly from keywords to this function?